kalite/contentload/management/commands/channels/khan.py

```diff
@@ -139,6 +139,12 @@ def build_full_cache(items, id_key="id"):
         except AttributeError:
             logging.error("Unable to serialize %r" % item)
 
+        item = whitewash_node_data(item)
+
+        if ids:
+            if item["id"] not in ids:
```

Comment:
Should this use `id_key` (or what's that for?) -- and is `ids` a blacklist? May be a better name for it?